ush/nems_configure.sh

```diff
@@ -48,6 +48,8 @@ else
   EPS_IMESH='1.0e-1'
 fi
 
+ATMTILESIZE=$(echo $CASE | cut -c2-)
```

Comment:
A variable with the same function is already available: $RESTILE (set in config.fcst). If the variable isn't currently available to this script, add an export there an switch to that variable.

Reply:
Thanks! I looked everywhere for an existing variable and didn't find one.